handlers/user_handler.go

```diff
@@ -43,8 +41,8 @@ func (h *Handler) AnonymousStatsHandler(w http.ResponseWriter, req *http.Request
 	if usageStats != "" {
 		aUsageStats, err := strconv.ParseBool(usageStats)
 		if err != nil {
-			err = errors.Wrap(err, "unable to parse anonymousUsageStats")
-			logrus.Error(err)
+			h.log.Error(err)
```

Comment:
A similar abstract functiuon for `PraseBool` error?

Reply:
So for this `ParseBool` error I need to create a new function right?

Reply:
I got a little confused in this `ParseBool` error can you tell me what I need to do here?